components/pages/Messages/Messages.js

```diff
@@ -188,11 +195,15 @@ export function Messages({ fromUserId }) {
                         {message.direction !==
                           threadData?.messages[nextMessage]?.direction &&
                         message.direction !== 'sent' ? (
-                          <img
-                            src={threadData?.from_user_thumbnail}
-                            alt="sender profile pic"
-                            className={styles.messagesSenderAvatar}
-                          />
+                          threadData.from_user_id && (
+                            <Image
+                              src={threadData?.from_user_thumbnail}
```

Comment:
We know threadData is not null here.
```suggestion
                              src={threadData.from_user_thumbnail}
```